src/Common/MetricHelper.cs

```diff
@@ -530,6 +531,7 @@ public class AzurePSQoSEvent
     public string SessionId { get; set; }
     public string SubscriptionId { get; set; }
     public string TenantId { get; set; }
+    public bool SurveyPrompted { get; set; }
```

Comment:
it should be nullable bool?. we need to add it into telemetry only when its value is true.

Reply:
no need for three states, add logic to add it only when value is true